### PR TITLE
Lab7

### PR DESCRIPTION
**Pull request by PassKeyRa**

### CODE REVIEW

**Review by boggda — APPROVED** [review]

File: monitoring/promtail/promtail.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
+server:
+  http_listen_port: 9080
+  grpc_listen_port: 0
+  log_level: "warn"
```
Comment (chermnyx) [inline monitoring/promtail/promtail.yml:4]: Why do you need the server of promtail?

File: monitoring/promtail/promtail.yml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            time: time
+      - timestamp:
+          source: time
+          format: RFC3339Nano
+      - regex:
+          expression: ^(?P<image_name>([^|]+))\|(?P<container_name>([^|]+))$
+          source: "tag"
+      - labels:
+          image_name:
+          container_name:
```
Comment (chermnyx) [inline monitoring/promtail/promtail.yml:42]: "output" part is not specified

File: monitoring/promtail/promtail.yml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+      - match:
+          # drop all logs for all containers that don't have --log-opt: tag: "{{:ImageName}}|{{:Name}}"
+          selector: '{job="docker",container_name="",image_name=""}'
+          action: drop
+
+      - match:
```
Comment (chermnyx) [inline monitoring/promtail/promtail.yml:49]: "timestamp" is not specified

File: monitoring/promtail/promtail.yml (hunk 1 above)
Comment (PassKeyRa) [reply monitoring/promtail/promtail.yml:4]: For debug purposes

**Pull request closed without merging**